lib/engine/game/base.rb

```diff
@@ -1065,6 +1065,12 @@ def cache_objects
       def bankruptcy_limit_reached?
         @players.any?(&:bankrupt)
       end
+
```

Comment:
not a big fan of assignment like this prefer entity: nil

entity ||= current_entity